### DISCUSSION TITLE
Manual start service ( via manual:yes ) get started when running initctl reload

### DISCUSSION THREAD

**Discussion by kenzanin** [finit-project/finit#180] [Q&A] [answered]
hi,

i have multiple service with tag manual:yes, and its working, my service is in stop status every time iam reboot my laptop, but when i use
sudo initctl reload 
all my manual service is started
iam using 
artix linux with finit master git
and here is one of my service command in /etc/finit/enabeled  
bftpd.conf
service manual:yes log bftpd -D -- bftpd service

is this a bug?

thanks

**Comment by troglobit** [comment, marked as answer]
Hmm, yeah that sounds like a bug.  Lemme check.

**Reply by troglobit** [reply]
Turns out this is a limitation of the original patch submitted by another user.  I'll attempt to patch it, limiting per-runlevel, so an `initctl reload` would not affect it, i.e., stopped and it remains stopped, started and it remains running. ~~However, changing runlevel would require manual start again.~~  Sounds OK?

Edit: changing runlevel and it will remain running if already started (or stopped if already stopped), unless condition or not in new runlevel.

**Reply by troglobit** [reply]
Fixed in 046ecfa.

**Reply by kenzanin** [reply]
thanks
